server/sonar-web/src/main/js/app/components/App.tsx

```diff
@@ -76,23 +88,18 @@ class App extends React.PureComponent<Props, State> {
   fetchAppState = () => {
     return this.props.fetchAppState().then(appState => {
       if (this.mounted) {
-        const onSonarCloud =
-          appState.settings !== undefined &&
-          appState.settings['sonar.sonarcloud.enabled'] === 'true';
         this.setState({
           branchesEnabled: appState.branchesEnabled,
           canAdmin: appState.canAdmin,
-          onSonarCloud
+          onSonarCloud: this.isOnSonarCloud(appState)
         });
       }
+      return appState;
     });
   };
 
-  finishLoading = () => {
-    if (this.mounted) {
-      this.setState({ loading: false });
-    }
-  };
+  isOnSonarCloud = (appState: any) =>
+    appState.settings !== undefined && appState.settings['sonar.sonarcloud.enabled'] === 'true';
```

Comment:
The check should be based on `areThereCustomOrganizations` no ? To now if we want to do something related to organizations or not, that's what we do nearly everywhere.

Reply:
Ok, I will request "my" organizations when org support is enabled, and I will keep the rest.